docker_kernel/magics/tag.py

```diff
@@ -35,24 +37,25 @@ def VALID_OPTIONS() -> dict[str, FlagDict]:
         }
     
     def _execute_magic(self) -> None:
-        target = self._args[0]
+        """
+        usage:
+        %tag source_image[:tag] target_image[:tag]
+        """
+        source_image = self._args[0]
+        target_image = self._args[1]
         try:
-            name, tag = target.split(":")
+            name, tag = target_image.split(":")
         except ValueError as e:
             # If no colon is provided the default tag is used
-            if ":" not in target:
-                name = target
-                tag = None
+            if ":" not in target_image:
+                name = target_image
+                tag = 'latest'
```

Comment:
Don't specify "latest" here. Just use `None` and let the `APIClient`  handle the rest in the kernel